test/parallel/test-tls-cli-min-version-1.3-max-version-1.2.js

```diff
@@ -0,0 +1,15 @@
+// Flags: --tls-min-v1.3 --tls-max-v1.2
+'use strict';
+const common = require('../common');
+if (!common.hasCrypto) common.skip('missing crypto');
+
+// Check behaviour when minimum is higher than maximum.
+
+const assert = require('assert');
+const tls = require('tls');
+
+assert.strictEqual(tls.DEFAULT_MAX_VERSION, 'TLSv1.2');
+assert.strictEqual(tls.DEFAULT_MIN_VERSION, 'TLSv1.3');
+
+// Check the min-max version protocol versions against these CLI settings.
+require('./test-tls-min-max-version.js');
```

Comment:
Not objecting, but all things being equal, I'd prefer this use a fixture or have the relevant code here in this file rather than loading another test file. This could potentially make this test brittle when that other test file is edited, and someone might not notice that they've broken this test while working on the other one (because they are likely to only run the test file they are editing).

Reply:
I'm looking at reworking the code to address Ben's comment, but please note that this is the pattern of all the existing TLS CLI tests. I think its a simple, readable, and effective organization, and that our CI catches missteps pretty quickly, but I've no objection in principle to someone doing a bulk refactor of all the tests using the pattern.